src/Admin/Designation/CandidateAdmin.php

```diff
@@ -119,12 +119,9 @@ protected function configureDatagridFilters(DatagridMapper $filter)
             ->add('quality', ChoiceFilter::class, [
                 'show_filter' => true,
                 'label' => 'Qualité',
-                'field_type' => ChoiceType::class,
+                'field_type' => TerritorialCouncilQualityEnum::class,
```

Comment:
`field_type` is not correct.

Reply:
nice shot 🔫 it should be `TerritorialCouncilQualityChoiceType`